doc/Makefile.am

```diff
@@ -196,12 +196,12 @@ geany-sciwrappers-gtkdoc-tmp.h: geany-sciwrappers-gtkdoc.h
 		-e 's,sci_,scintilla_object__GI__MARK_,g' \
 		$< > $@
 
-GeanyScintilla-1.0.gir: GeanyScintilla-1.0-pre.gir
+GeanyScintilla-1.0.gir: GeanyScintilla-1.0-tmp.gir
 	$(AM_V_GEN) cat $< \
```

Comment:
I think I already know the answer, but just in case, why not pipe `g-ir-scanner`'s output directly through `sed`, bypassing any intermediate files? It writes to stdout by default.